App.js

```diff
@@ -1,23 +1,64 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+'use strict';
+
+import { NavigatorIOS, StyleSheet, Text, View, Platform } from 'react-native';
+import React, { Component } from 'react';
+import { Font } from 'expo';
+import { NativeRouter, Route, Link } from 'react-router-native'
+import NavigationBar from 'react-native-navigation-bar';
+
+import HomePageIOS from './HomePageIOS';
+import HomePageAndroid from './HomePageAndroid';
+
+export default class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { fontLoaded: false };
+  }
+  async componentDidMount() {
+    await Font.loadAsync({
+      'helvetica-neue': require('./resources/fonts/HelveticaNeueLt.ttf')
+    });
+    this.setState({ fontLoaded: true });
+  }
 
-export default class App extends React.Component {
   render() {
+    if (Platform.OS === 'ios')
+    return (
+      this.state.fontLoaded ? (
+        <NavigatorIOS
```

Comment:
Are you sure you can't also use the `NativeRouter` here instead of NavigatorIOS?